### PR TITLE
Added support to check-rds-events to do instance specific checks

### PR DESCRIPTION
**Pull request by jessechahal**
previous version would look at every single available rds instance in a region, not too friendly for people who have 100's of rds instances up and running. I've modified this to allow checks for single rds instances in a manner comparable to check-rds.rb.

### CODE REVIEW

File: bin/check-rds-events.rb [hunk 1]
```diff
@@ -20,7 +20,14 @@
 #   gem: sensu-plugin
 #
 # USAGE:
-#  ./check-rds-events.rb -r ${your_region} -k ${your_aws_secret_access_key} -a ${your_aws_access_key}
+#  Check's a specific RDS instance on a specific availability zone for critical events
+#  check-rds-events.rb -r ${your_region} -k ${your_aws_secret_access_key} -a ${your_aws_access_key} -i ${your_rds_instance_id_name}
+#
+#  Checks all RDS instances on a specific availability zone
+#  check-rds-events.rb -r ${your_region} -k ${your_aws_secret_access_key} -a ${your_aws_access_key}
```
Comment (analytically) [inline bin/check-rds-events.rb]: where do I specify the zone? region?

File: bin/check-rds-events.rb [hunk 2]
```diff
@@ -52,6 +59,11 @@ class CheckRDSEvents < Sensu::Plugin::Check::CLI
          description: 'AWS Region (defaults to us-east-1).',
          default:     'us-east-1'
 
+  option :db_instance_id,
+         short:       '-i N',
+         long:        '--db-instance-id NAME',
+         description: 'DB instance identifier'
```
Comment (analytically) [inline bin/check-rds-events.rb]: Can you expand a little on this?

File: bin/check-rds-events.rb [hunk 3]
```diff
@@ -72,8 +84,18 @@ def maint_clusters
     rds = AWS::RDS::Client.new aws_config
 
     begin
-      # fetch all clusters identifiers
-      clusters = rds.describe_db_instances[:db_instances].map { |db| db[:db_instance_identifier] }
+      if !config[:db_instance_id].nil? && !config[:db_instance_id].empty?
+        db_instance = rds.describe_db_instances(db_instance_identifier: config[:db_instance_id])
+        if db_instance.nil? || db_instance.empty?
```
Comment (analytically) [inline bin/check-rds-events.rb]: can this return empty?

File: bin/check-rds-events.rb (hunk 1 above)
Comment (jessechahal) [inline bin/check-rds-events.rb]: I don't believe you can specify the zone, you can only specify the region. This was the original behaviour and original documentation format that existed in the check. I should change the comment above though to not state `specific availability zone` and instead state `specific region`

File: bin/check-rds-events.rb (hunk 2 above)
Comment (jessechahal) [inline bin/check-rds-events.rb]: Is there a particular part you would like me to expand upon? Should I increase the the content of the description? The option here is actually copy/paste from the only other rds check `check-rds.rb`. I figured consistency between the two would reduce confusion .

File: bin/check-rds-events.rb (hunk 3 above)
Comment (jessechahal) [inline bin/check-rds-events.rb]: yes, you can search for anything you like but that doesn't mean the rds instance actually exists. We should check the returned object to see if we actually got something back from amazon.

**Comment by jessechahal** [comment]
ok so what happened with the travis-ci builds? It looks like they are failing for multiple PR's and have to do with ruby env variables. Is ruby installation not set on the env?

**Comment by sstarcher** [comment]
@jessechahal the travis ci builds should be fixed now.  If you rebase off master it should go green.

**Pull request merged**